FurEver/Pages/DogQuiz.cshtml

```diff
@@ -0,0 +1,50 @@
+﻿@page
```

Comment:
Is this a net-new page as part of the code review?  Everything is coming up green, which indicates it was not present in the branch prior to this commit.